src/utils/entitySpec.js

```diff
@@ -65,6 +65,12 @@ export default class EntitySpec {
     setIfValue(this, 'changes', changes)
   }
 
+  get url() {
```

Comment:
This should likely be a `toUrl` method in the same way that we have `toPath` and `toString`. `url` is not really a property of an entity spec, rather a format. 

Also, feels funny to have this on `EntitySpec` and have it only support github. 

If you just want to support just the source location scenario then perhaps put this method somewhere else (more specific to that case)

It seems that the prop is only used in 2 places. One it explicitly has to strip off the accessor and put in a real value. the other is for the Section title and I suggested above that that go away. More support for not having this here and not having it as an accessor.

Reply:
thoughts on this?

Reply:
Having it like this allows to handle objects from the backend isomorphically to these. Some render code is sometimes using raw backend objects, sometimes these. I immediately ran into bugs when trying to change `get url` into a `toUrl` method, so I would really like to first write some unit and integration tests before refactoring like that.

Reply:
Having it like this allows to handle objects from the backend isomorphically to these. Some render code is sometimes using raw backend objects, sometimes these. I immediately ran into bugs when trying to change `get url` into a `toUrl` method, so I would really like to first write some unit and integration tests before refactoring like that.